doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -13,10 +13,10 @@ For detailed information about supported functionalities, see the official `Fast
 
 .. note::
    The Fast Pair support in the |NCS| is :ref:`experimental <software_maturity>`.
-   The implementation is not yet ready for production and extensions are not supported.
+   The implementation is not yet ready for production and extensions are not fully supported.
 
-   The implementation does not pass end-to-end integration tests in the Fast Pair Validator.
-   The procedure triggered in Android settings is successful (tested with Android 10).
+   The implementation passes Fast Pair Validator tests.
```

Comment:
I think you should mention that you used beta version of the Validator (non-beta would not work)